src/routes/add-article.ts

```diff
@@ -24,8 +24,8 @@ export default (): AppMiddleware => (
       throw new createHttpError.BadRequest(`Article must have a blank node identifier (${termToString(id)} given)`);
     }
 
-    if (request.dataset.match(id, schema('name')).size === 0) {
-      throw new createHttpError.BadRequest(`Article must have at least one ${termToString(schema('name'))}`);
+    if (request.dataset.match(id, schema.name).size === 0) {
+      throw new createHttpError.BadRequest(`Article must have at least one ${termToString(schema.name)}`);
```

Comment:
As `name` is now a defined property on the builder, TypeScript now correctly recognises it as a `NamedNode`.

Though PHPStorm seems a bit confused (though doesn't complain):
![Screenshot 2020-02-28 at 10 53 33](https://user-images.githubusercontent.com/1784740/75543861-aa3d1200-5a1a-11ea-904d-bbeabf122aad.png)